worlds/sc2/__init__.py

```diff
@@ -395,6 +400,16 @@ def fill_item_pool_with_dummy_items(self: SC2World, locked_locations: List[str],
         pool.append(item)
 
 
+def place_local_victory_items(multiworld: MultiWorld, world: World, pool: list):
+    victory_locations = [location for location in multiworld.get_unfilled_locations(world.player)
+                         if location.name.endswith((": Victory", ": Defeat"))]
```

Comment:
The endswith is used elsewhere, should be wrapped into a function (`is_victory_location`)